Gemfile.lock

```diff
@@ -1058,6 +1058,7 @@ DEPENDENCIES
   sidekiq-pro!
   sidekiq-scheduler (~> 3.0)
   simplecov (< 0.18)
+  slack-notify
```

Comment:
I _think_ this isn't mean to be here

Reply:
ok thanks, i removed it, it gets readded every time i run bundler though so maybe it should be in master?